core/src/main/java/com/linecorp/armeria/server/DefaultRoute.java

```diff
@@ -180,6 +179,20 @@ public RoutingResult apply(RoutingContext routingCtx, boolean isRouteDecorator)
         return builder.build();
     }
 
+    private void deferStatusException(RoutingContext routingCtx, HttpStatus httpStatus) {
+        if (isFallback) {
+            // Do not defer an exception if this route is a fallback route, which is matched
+            // only when no configured route was matched.
+            //
+            // For example, assume that a route '/a/b/c/' supports HTTP GET method only.
+            // Its fallback route would be added as a path of '/a/b/c' with supporting HTTP GET method as well.
+            // In this case, '404 not found' would make sense rather than '405 method not allowed'
+            // if a 'DELETE /a/b/c' request is received, because the fallback route radically does not exist.
```

Comment:
What does radically here mean?

Reply:
I meant the route wasn't specified by a user. So would it be better if I rewrite it as follows?

`because the fallback route wasn't specified by a user.`